exporter/exporterhelper/retry_sender.go

```diff
@@ -39,6 +39,28 @@ type RetrySettings struct {
 	MaxElapsedTime time.Duration `mapstructure:"max_elapsed_time"`
 }
 
+func (cfg *RetrySettings) Validate() error {
+	if !cfg.Enabled {
+		return nil
+	}
+	if cfg.InitialInterval < 0 {
+		return errors.New("initial interval must be non-negative")
```

Comment:
I'm not sure how consistently we follow this or not, but typically I advise that we use the exact keys which the user would need to work with. If this goes against established patterns, feel free to ignore. Otherwise, this could be applied to all error messages below as well.
```suggestion
		return errors.New("'initial_interval' must be non-negative")
```